docs/guides/integration-options-private-operator.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Private Operator Integrations
+description: Overview of UID2 Private Operator options.
+hide_table_of_contents: false
+sidebar_position: 02
+---
+
+import Link from '@docusaurus/Link';
+
+# Private Operator Integrations
+
+A Private Operator is a private instance of the UID2 <Link href="../ref-info/glossary-uid#gl-operator">Operator</Link>. This means that a specific entity hosts a private instance of the UID2 Operator, exclusively for their own use.
+
+A Cloud-based Private Operator runs in an <Link href="../ref-info/glossary-uid#gl-enclave">enclave</Link>&#8212;a memory-hardened virtual machine that prevents unauthorized access, so that unauthorized individuals, even if they are administrators, cannot download any configuration information or data from the virtual machine.
+
+This is an extra layer of security so that no one can see the salt values used to produce raw UID2s, and instead only the salt bucket ID is stored outside the Operator.
+
+Becoming a Private Operator includes several additional steps, and uses resources that the participant must provide.
+
+On this page, you'll find a high-level overview of integration steps and integration options, with links to additional information for each option.
+
+## Private Operator Requirements
+
+The participant must host, configure, maintain, and update the Private Operator instance, and must conform to strict security measures. Engineering resources are required to integrate and to make ongoing updates.
+
+The participant must sign a contract (see [Account Setup](../getting-started/gs-account-setup.md)) to host a Private Operator instance.
+
+:::note
+A Private Operator has no visibility into the raw UID2s or UID2 tokens processed by a Public Operator or another Private Operator. Each Private Operator is a completely closed infrastructure.
+:::
+
+## Private Operator Workflow
+
+Every Private Operator runs in an enclave (AWS), confidential space (GCP), or confidential computing environment, (Azure). Each of these ensures that the Private Operator runs in a protected memory space.
+
+The basic workflow is as follows:
+
+1. On startup, the Private Operator goes through an attestation process with the Core service. The attestation process verifies that the Operator is running in a secure and memory-hardened environment that is known and trusted, and that the environment hasn't been tampered with.
+
+1. When the Operator passes that attestation process, the Core service gives the Private Operator a secure S3 URL for retrieving the information it needs for startup.
```

Comment:
There is more than one URL returned, so maybe: the Core service gives the Private Operator secure S3 URLs for ....

Reply:
Done, thx.